src/core/state.py

```
@@ -208,62 +290,114 @@ def record_execution(self, job_id: str, seq_id: str, execution_result: Dict[str,
+                        'seq_run_timeout': int(job_data.seq_run_timeout) if job_data.seq_run_timeout is not None else 300,
+                        'run_state': job_data.run_state.value if hasattr(job_data.run_state, 'value') else str(job_data.run_state),
+                        'last_run': job_data.last_run.isoformat() if job_data.last_run else None,
+                        'next_run': job_data.next_run.isoformat() if job_data.next_run else None,
+                        'latest_result': self._clean_for_json(job_data.latest_result),
+                        'error_logs': self._clean_for_json(job_data.error_logs),
+                        'parameters': self._clean_for_json(job_data.parameters),
+                        'seq_run_interval': job_data.seq_run_interval,
+                        'start_date': job_data.start_date.isoformat() if job_data.start_date else None,
+                        'end_date': job_data.end_date.isoformat() if job_data.end_date else None,
+                    }
+                    print(f"DEBUG: Manual serialization successful, using manual dict")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)